lib/Serialization/ModuleFile.h

```diff
@@ -623,7 +623,7 @@ class ModuleFile
   /// Load the methods within the given class that produce
   /// Objective-C class or instance methods with the given selector.
   ///
-  /// \param classDecl The class in which we are searching for @objc methods.
+  /// \param typeDecl The class in which we are searching for @objc methods.
   /// The search only considers this class and its extensions; not any
   /// superclasses.
   ///
```

Comment:
Could you as well replace occurrences of "class" (where it refers to a type declaration) with "type" throughout this documentation block to match the new parameter name? 

P.S. There may be other `loadObjCMethods` declarations that need similar corrections if you are interested in auditing them.

Reply:
For sure. I'll get on that.